### PR TITLE
First batch of GDI SOP documentation

### PR DESCRIPTION
**Pull request by M-casado**
Along the shift from Google Docs to GitHub, I grouped in this PR three main changes:

- [ZenHub#255](https://app.zenhub.com/workspaces/t43-gdi-sops-662a0b0fe38b0405fb600d67/issues/zh/255): **Define object accessioning proposal in GitHub**. This proposal is up to review, given that there are some changes, out of which I would highligh:
   - Filenames of GDI SOPs only contain the base identifier (e.g., ``GDI-SOP0001_<title>.md``), rather than the version or node's code (e.g., ``GDI-SOP0001.v1_SWE.v1_<title>.md``). See details within the accessioning proposal.
- [ZenHub#256](https://app.zenhub.com/workspaces/t43-gdi-sops-662a0b0fe38b0405fb600d67/issues/zh/256): **Create GitHub issue template**: request new SOP.
   - Through this quick and user-friendly template, anyone can request the creation of a new SOP without knowing technical details nor how to use GitHub (beyond creating an issue, of course).
- [ZenHub#262](https://app.zenhub.com/workspaces/t43-gdi-sops-662a0b0fe38b0405fb600d67/issues/zh/262): **Add General SOP template to GitHub repo**. Highlighted changes:
   - I modified some parts of the template, added more descriptions and examples.
   - I added a metadata section at the beginning, where we can track the metadata of the files, rather than in the filenames.

### CODE REVIEW

**Comment by bhendriksze** [comment]
Seems to cover all that was discussed before. No review comments.

File: .github/ISSUE_TEMPLATE/new_sop_request.yaml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        ## New SOP Request
+
+  - type: markdown
+    attributes:
+      value: |
+        This form serves as a template for **new GDI SOP requests**. It can be used to request a standard operation procedure (SOP) that could help the GDI members to execute a process in a standardized way.
+        Thank you for contributing to the Genomics Data Infrastructure (GDI) SOPs!
```
Comment (mattiasstromberg) [inline .github/ISSUE_TEMPLATE/new_sop_request.yaml:18]: Replace "Genomics" with "Genomic".

File: docs/GDI-SOP_github-management.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 The GDI SOP repository ([**GenomicDataInfrastructure/standard-operating-procedures**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures)) contains both European-level SOP _instances_ and Node-specific SOP _templates_. In order to adapt the Node-specific SOP templates, GDI nodes are expected to make a copy of this GitHub repository and modify them to suit them to their needs.
 
 Given that the base repository is public, forks of this repository cannot be private. Therefore, making a node's copy of the repository can be done in different ways, explained in this document, depending on the level of privacy that each node may require:
-- **Public node's repository**. If the node does not require for a private repository, copying the source repository is straightforward. See section _[1. Public repository](./github-management.md#1-public-repository)_.
```
Comment (mattiasstromberg) [inline docs/GDI-SOP_github-management.md:5]: Replace "require for" with "require".

File: docs/GDI-SOP_github-management.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
...
-- **Private node's repository**. On the contrary, if nodes need their SOP instances to be private (e.g. for security reasons), an alternative route is taken. See section _[2. Private repository](./github-management.md#2-private-repository)_.
+- **Public node's repository**. If the node does not require for a private repository, copying the source repository is straightforward. See section _[1. Public repository](#1-public-repository)_.
```
Comment (mattiasstromberg) [inline docs/GDI-SOP_github-management.md:5]: Replace "require for" with "require".

File: docs/GDI-SOP_sop-accessioning.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
+# GDI SOP Accessioning Proposal
+## Introduction
+This document outlines the **accessioning system for Standard Operating Procedures** (SOPs) within the Genomics Data Infrastructure (GDI) project. It covers the SOP identifier format, file naming conventions, automation workflows, repository structure, version control, and referencing mechanisms. This system aims to streamline the management, versioning, and referencing of SOPs, ensuring clarity, consistency, and ease of use across the project.
```
Comment (mattiasstromberg) [inline docs/GDI-SOP_sop-accessioning.md:3]: Replace "Genomics" with "Genomic".

File: docs/GDI-SOP_sop-accessioning.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+1. [Identifier Format](#identifier-format)
+1. [File Naming Conventions](#file-naming-conventions)
+1. [SOP Repository Structure](#sop-repository-structure)
+1. [SOP Life-cycle](#sop-life-cycle)
+1. [Automatic SOP accessioning](#automatic-sop-accessioning)
+1. [Version Control and Updates](#version-control-and-updates)
+
+## Identifier Format
+In this section we cover how identifiers for SOP-related documents are given. The main two categories are European-level and Node-specific SOPs.
```
Comment (mattiasstromberg) [inline docs/GDI-SOP_sop-accessioning.md:15]: Replace "main two" with "two main".

**Review by FracassandoCasualmente — APPROVED** [review]
Looks good to me. Great work!

**Pull request merged**